db/models/flags_v2/assessment_flag.py

```diff
@@ -0,0 +1,57 @@
+from uuid import uuid4
+
+from db import db
+from db.models.flags_v2.flag_update import FlagStatus
+from flask_sqlalchemy import DefaultMeta
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import String
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import relationship
+
+BaseModel: DefaultMeta = db.Model
+
+
+class AssessmentFlag(BaseModel):
+
+    # def __init__(self, id, application_id, latest_status, latest_allocation, sections_to_flag, updates):
+    #     self.id = id
+    #     self.application_id = application_id
+    #     self.latest_allocation = latest_allocation
+    #     self.latest_status = latest_status
+    #     self.sections_to_flag = sections_to_flag
+    #     self.updates = updates
+
+    def __init__(
+        self,
+        application_id,
+        latest_status,
+        latest_allocation,
+        sections_to_flag,
+        updates,
+        **kwargs,
+    ):
+        self.application_id = application_id
+        self.latest_allocation = latest_allocation
+        self.latest_status = latest_status
+        self.sections_to_flag = sections_to_flag
+        self.updates = updates
+        if "id" in kwargs:
+            self.id = kwargs["id"]
```

Comment:
Do we need to call the super `__init__` too at all? Or does this not cause any issues as far as you know?

Reply:
This all works as-is at the moment. I've created a tech debt ticket to start using the sqlalchemy dataclass integration when we upgrade ot sqlalchemy 2.0 as per Adam's comment above.